lib/internal/Magento/Framework/Interception/PluginList/PluginList.php

```diff
@@ -168,15 +168,15 @@ protected function _inheritPlugins($type)
             if (is_array($plugins) && count($plugins)) {
                 $this->filterPlugins($plugins);
                 uasort($plugins, [$this, '_sort']);
-                $this->trimInstanceStartingBackslash($plugins);
-                $this->_inherited[$type] = $plugins;
                 $lastPerMethod = [];
                 foreach ($plugins as $key => $plugin) {
                     // skip disabled plugins
                     if (isset($plugin['disabled']) && $plugin['disabled']) {
```

Comment:
We have tried to test this but could not recreate the bug on 2.2. Every time you setup a plugin as disabled you will hit this if and then unset the plugin.

@scottsb Is there a testing case that I am missing here.

Reply:
I'll put together a sample test case next week.